e2e/test/scenarios/question/notebook.cy.spec.js

```diff
@@ -630,6 +630,7 @@ describe("scenarios > question > notebook", { tags: "@slow" }, () => {
 
       visualize();
 
+      cy.findByLabelText("Switch to data").click();
```

Comment:
After I fixed field metadata, we know more about fields, so we show a bar chart here instead of a table like previously, so we need to manually switch to the table view.